pr2eus/robot-interface.l

```diff
@@ -1236,17 +1236,18 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
 ;;
 ;; for text-to-speech services
 (defmethod robot-interface
-  (:play-sound (sound &key arg2 (topic-name "robotsound") wait)
-   (funcall #'play-sound sound :arg2 arg2 :topic-name topic-name :wait wait))
-  (:speak (text &key (lang "") (topic-name "robotsound") wait)
+  (:play-sound (sound &key arg2 (topic-name "robotsound") wait (volume 1.0))
```

Comment:
how about `(:play-sound (sound &rest args &allow-other-keys) (apply #'playsound sound args)(` or `(:play-sound (&rest args) (apply #'play-sound args))` ?

Reply:
thank you for your review.
I update to pass all the args to `play-sound` with `apply`